inst/rmarkdown/templates/mortality/skeleton/skeleton.Rmd

```diff
@@ -1178,19 +1349,31 @@ to change this, set proptotal = FALSE
 
 ```{r describe_by_age_group_and_sex}
```

Comment:
@pbkeating but for example in this table - the gtsummary version is much simpler